.kitchen.dokken.yml

```diff
@@ -68,4 +68,4 @@ platforms:
     pid_one_command: /bin/systemd
     intermediate_instructions:
       - RUN /usr/bin/apt-get update
-      - RUN /usr/bin/apt-get install apt-transport-https lsb-release procps net-tools -y
+      - RUN /usr/bin/apt-get install apt-transport-https lsb-release procps net-tools logrotate -y
```

Comment:
it comes installed by default on the full OS but not the container.
See https://travis-ci.org/sous-chefs/mongodb/jobs/234474765#L461